gauge_api_steps/substitute.py

```diff
@@ -79,5 +80,13 @@ def _evaluate_expression(expression: str) -> str:
         file_path = assert_file_is_in_project(file_name)
         with open(file_path, 'r') as f:
             return f.read()
+    elif expression_lower.startswith(("gql:", "graphql:")):
```

Comment:
Can those branches move into private methods? What if there are other expressions to be considered in the future? The length of the method will grow more.